frontend/src/metabase/query_builder/components/filters/modals/BulkFilterList/BulkFilterList.styled.tsx

```diff
@@ -1,26 +1,29 @@
 import styled from "@emotion/styled";
 import { color } from "metabase/lib/colors";
-import {
-  space,
-  breakpointMinHeightMedium,
-} from "metabase/styled-components/theme";
+import { space, breakpointMinSmall } from "metabase/styled-components/theme";
 
 export const ListRoot = styled.div`
   margin-bottom: 1rem;
 `;
 
 export const ListRow = styled.div`
-  padding: 1.5rem 3rem;
-  ${breakpointMinHeightMedium} {
-    padding: 2.5rem 3rem;
-  }
+  padding: 1.5rem 2rem;
   border-bottom: 1px solid ${color("border")};
   &:last-of-type {
     border-bottom: none;
   }
+  &:hover,
+  :focus-within {
+    background-color: ${color("bg-light")};
+  }
```

Comment:
adding a hover state makes the natural behavior of scanning with your mouse more interactive